data/Sri Guru Granth Sahib Ji/0092.json

```diff
@@ -339,7 +339,7 @@
         "translations": {
           "English": {
             "Dr. Sant Singh Khalsa": {
-              "translation": "O my Lord, please save me!||1||Pause||",
+              "translation": "O my Lord, please save me!\" ||1||Pause||",
```

Comment:
It’s a bit weird that quotes can span multiple lines

Reply:
Yep
![](https://i.imgur.com/vNW2VWP.png)